exe/facter.cc

```diff
@@ -264,6 +264,9 @@ int main(int argc, char **argv)
         bool show_legacy = vm.count("show-legacy");
         facts.write(boost::nowide::cout, fmt, queries, show_legacy);
         boost::nowide::cout << endl;
+    } catch (locale_error const& e) {
+        cerr << "Failed to initialize logging system due to a locale error: " << e.what() << endl;
```

Comment:
I'm not sure if we can use nowide at this point, since our locale is potentially hosed. colorize ought to work, though